build_docs

```diff
@@ -103,8 +104,7 @@ def run_build_docs(args):
                 '-v',
                 '%s:/doc/%s:ro,cached' % (repo_root, repo_name)
             ])
-        build_docs_args.append(
-                '/doc/' + repo_name + path.replace(repo_root, ''))
+        return '/doc/' + repo_name + path.replace(repo_root, '')
```

Comment:
I dropped automatically adding it to the list because we don't want to do that for `--sub_dir`.